schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -48,2267 +48,15 @@
       "description": "User defined functions"
     },
     "resources": {
```

Comment:
I take it you've passed the structural changes by Stephen? Are changes to the ARM Tools extension needed before this PR is merged?